nmrfx-bom/pom.xml

```
@@ -19,6 +19,11 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.nanalysis</groupId>
+                <artifactId>jcamp-parser</artifactId>
+                <version>1.0</version>
```

This needs to change to get the fix about the module name:
```suggestion
                <version>1.0.1</version>
```

---

Related PR: 
https://github.com/nanalysis/jcamp-parser/pull/3

For reference, I put the built artifact there:
https://github.com/nanalysis/maven-repository/tree/public/com/nanalysis/jcamp-parser/1.0.1

Which is then referenced as an additional maven repository in our projects. We should not replace released versions, only add new ones. This is open sourced, other people could depend on it.